http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/FollowupComponent.scala

```diff
@@ -0,0 +1,43 @@
+package org.broadinstitute.dsde.workbench.leonardo
+package db
+
+import scala.concurrent.ExecutionContext
+import LeoProfile.api._
+import LeoProfile.mappedColumnImplicits.statusMappedColumnType
+import org.broadinstitute.dsde.workbench.leonardo.model.google.ClusterStatus
+import org.broadinstitute.dsde.workbench.leonardo.monitor.LeoPubsubMessage.ClusterFollowupDetails
+
+case class FollowupRecord(clusterId: Long, status: ClusterStatus, masterMachineType: Option[String])
+
+class FollowupTable(tag: Tag) extends Table[FollowupRecord](tag, "CLUSTER_FOLLOWUP") {
+  def clusterId = column[Long]("clusterId", O.PrimaryKey)
+  def status = column[ClusterStatus]("status", O.Length(254))
```

Comment:
I asked this in another place, but does this table need a `status` column? Seems it's duplicating `CLUSTER.status`, to which it has a 1-1 relationship.

Reply:
this status is needed, as otherwise subscriber has no idea at what cluster status it should perform the associated follow-up action

Reply:
I see - I didn't realize that nuance, I think that makes sense then